### PR TITLE
Minor fixes

### LINKED ISSUE #198: EconThermal values can't be serialized to JSON
The struct defines variablecost as Union{Function,Array{Tuple{Float64, Float64}}}.  A value with a Function cannot be serialized.  It needs to be converted before being serialized.

### PR DESCRIPTION
**Pull request by jd-lara**
Fixes #198 and corrects tests for a more consistent data collection

@daniel-thom this should address the serialization problem

### CODE REVIEW

**Comment by jd-lara** [comment]
@claytonpbarrows this PR is in draft mode, not ready to merge

**Comment by claytonpbarrows** [comment]
@jd-lara I saw that after I hastily clicked a button

**Comment by jd-lara** [comment]
I am getting some test errors that are also addressed in #208.

File: src/parsers/pm2ps_parser.jl [hunk 1]
```diff
@@ -223,21 +228,22 @@ function make_thermal_gen(gen_name, d, bus)
...
-            cost = x-> d["cost"][1]*x + d["cost"][2]
+            cost = (0.0, d["cost"][1])
+            fixedcost = d["cost"][2]
         elseif d["ncost"] == 3
-            cost = x-> d["cost"][1]*x^2 + d["cost"][2]*x + d["cost"][3]
+            cost = (d["cost"][1], d["cost"][2]) 
+            fixedcost = d["cost"][3]
         elseif d["ncost"] == 4
-            cost = x-> d["cost"][1]*x^3 + d["cost"][2]*x^2 + d["cost"][3]*x + d["cost"][4]
+            @error("PowerSystems currently can only support polynomial functions upto second degree")
```
Comment (daniel-thom) [inline src/parsers/pm2ps_parser.jl]: Should this throw an error like in the else clause?

Comment (jd-lara) [reply src/parsers/pm2ps_parser.jl]: fixed via 460e04f

File: Project.toml [hunk 2]
```diff
@@ -20,4 +19,4 @@ YAML = "ddb6d928-2868-570f-bddf-ab3f9cf99eb6"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [targets]
-test = ["Test"]
+test = ["Test"]
```
Comment (daniel-thom) [inline Project.toml]: How is it that there is no newline at the end of the file?

Comment (jd-lara) [reply Project.toml]: This is automatically generated, I don't know. I can add it manually

**Comment by daniel-thom** [comment]
Looks good to me.

**Pull request merged**